dev/archery/archery/crossbow/reports.py

```diff
@@ -30,6 +30,17 @@
 # TODO(kszucs): use archery.report.JinjaReport instead
 class Report:
 
+    ROW_HEADERS = [
+        "Task name",
```

Comment:
We could also use names like `"task_name"` instead (lowercase, no spaces), which are generally easier to use when afterwards playing with the data with pandas or R or something alike

Reply:
done! Thanks!